src/transformers/tape.js

```diff
@@ -78,8 +78,6 @@ const tPropertiesUnsupported = new Set([
     'notDeepLooseEqual',
     'notLooseEqual',
     'notLooseEquals',
-
-    'skip',
```

Comment:
There is a difference between test.skip and t.skip (t is the argument you get from Tape when running a test function). 

Can you add a test case asserting a warning for using t.skip? 

Thanks! : )